lib/board.rb

```diff
@@ -0,0 +1,21 @@
+class Board 
+
+   WIN_COMBINATIONS = [
+    [0, 1, 2], [3, 4, 5], [6, 7, 8],
+    [0, 3, 6], [1, 4, 7], [2, 5, 8],
+    [0, 4, 8], [2, 4, 6]
+  ]
+  
+  def initialize
+  @board = Array.new(9, " ")
+  end
+
+  def display_board
```

Comment:
As project requirement lib folder(logic) should not contain any user interaction like (no “puts” and “get.chomp” !). all UI should be in bin, so move All UI to bin and consider this change where necessary in this project.